### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pkankaras**
Read me file updated with additional link to the Plutus community playgrounds.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,8 +69,19 @@
 - [`PlutusTx.Data`](https://github.com/input-output-hk/plutus/blob/master/plutus-tx/src/PlutusTx/Data.hs), contains the definition of the `Data` type.
 - [`PlutusTx.IsData.Class`](https://github.com/input-output-hk/plutus/blob/master/plutus-tx/src/PlutusTx/IsData/Class.hs), defines the `IsData` class.
 
+## Plutus community playgroud
```
Comment (brunjlar) [inline README.md:72]: playgouNd

Comment (brunjlar) [reply README.md:72]: Please fix typo.

**Review by brunjlar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (pkankaras) [reply README.md]: fixed

**Review by pkankaras — COMMENTED** [review]
fixed typo.

**Pull request merged**